### ISSUE TITLE
Possible issue with CIMClassName and __str__

### ISSUE THREAD

**Issue by KSchopmeyer** [pywbem/pywbem#943]
Not sure if it is an issue or not but I had a problem Saturday where when I merged last master commit in I got a failure where I was trying to test against /:myclassname because I was using the string result of
CIMClassname.  Did this not previously return just the classname and if so, are users expecting this different behavior for the __str__ output of CIMClassName.

Right now this is just a question for discussion. I bring this up because it is an easy mistake to make for users that are trying to get the classname string out of CIMClassname.

**Comment by andy-maier** [comment]
Karl, `CIMClassName.__str__()` is documented to return the WBEM URI of the class path it represents. We changed the WBEM URIs to be conformant to DSP0207, which adds `/:` in front of the class name.

Since we now introduced the `to_wbem_uri()` methods on `CIMClassName` and `CIMInstanceName`, I guess we could change the meaning of `CIMClassName.__str__()` to be the class name, and thus to be compatible.

Discussion point.
